### PR TITLE
HTBHF-2676 Create CombinedIdentityAndEligibilityResponse for eligibility…

### PR DESCRIPTION
**Pull request by mikeSmith08**
… service to respond with.

Also update test data factory for IdentityAndEligibilityResponse according to main usage in claimant-service, this will mean we can remove the test data factory code from claimant-service.

### CODE REVIEW

File: common_rest/src/main/java/uk/gov/dhsc/htbhf/eligibility/model/CombinedIdentityAndEligibilityResponse.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@Builder(toBuilder = true)
+@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+public class CombinedIdentityAndEligibilityResponse {
+
+    @JsonProperty("dwpIdentityAndEligibilityResponse")
+    private final IdentityAndEligibilityResponse identityAndEligibilityResponse;
+
+    @JsonProperty("hmrcHouseholdIdentifier")
+    private final String hrmcHouseholdIdentifier;
+
```
Comment (YetAnotherMatt) [inline common_rest/src/main/java/uk/gov/dhsc/htbhf/eligibility/model/CombinedIdentityAndEligibilityResponse.java:52]: Can we consider in-lining the IdentityAndEligibilityResponse? Maybe we should have a chat about it after stand-up

**Review by dwybourn — DISMISSED** [review]

**Review by gethyn1 — DISMISSED** [review]

File: common_rest/src/main/java/uk/gov/dhsc/htbhf/eligibility/model/CombinedIdentityAndEligibilityResponse.java (hunk 1 above)
Comment (mikeSmith08) [reply common_rest/src/main/java/uk/gov/dhsc/htbhf/eligibility/model/CombinedIdentityAndEligibilityResponse.java:52]: Have inlined as we discussed.

File: common_test/src/main/java/uk/gov/dhsc/htbhf/TestConstants.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package uk.gov.dhsc.htbhf;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class TestConstants {
```
Comment (mikeSmith08) [inline common_test/src/main/java/uk/gov/dhsc/htbhf/TestConstants.java:6]: I want to start moving the TestConstants out of the dwp-api module into the common-test module as they are independant of the dwp-api module really, but will do in a separate PR. These constant will then be more accessible.

**Review by YetAnotherMatt — APPROVED** [review]
